stylesheets/styles.css

```
@@ -333,11 +333,17 @@ section {
 /* Footer */
 
 footer {
-  width:170px;
+  margin: 0 auto;
+  width: 960px;
   float:left;
   position:fixed;
-  bottom:10px;
-	padding-left: 50px;
+  bottom:0px;
```

別の修正があるのでついでに :pray: 0 は px 指定なしでも可ですー (上のmarginでも 0 だけの記述例がある) 
```suggestion
  bottom: 0px;
```

実は float: left もいらないかもしれないけど、CSS力が弱くて確証がないです… :bow:

---

[f03942b]
`float: left`確かに不要でした…！
他にも、marginの設定も今のこのシンプルな footer では不要だったので削除をしました(｀･ω･´)